src/gui/optionsdialog.ui

```diff
@@ -178,24 +163,36 @@
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
-                   <width>200</width>
+                   <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
-               <item row="1" column="0">
-                <widget class="QLabel" name="label_15">
-                 <property name="text">
-                  <string>Theme:</string>
+               <item row="1" column="0" colspan="4">
+                <widget class="QGroupBox" name="checkUseCustomTheme">
+                 <property name="title">
+                  <string>Use Custom Theme (Requires restart)</string>
+                 </property>
+                 <property name="checkable">
+                  <bool>true</bool>
                  </property>
+                 <layout class="QFormLayout" name="formLayout">
```

Comment:
use QHBoxLayout instead and don't worry about the path looking weird in Qt desginer/creator.